packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -179,6 +179,14 @@ const runMigrationCommand: CliCommandDefinition<CreateFlags> = {
       return
     }
 
+    output.print(
+      `\n${chalk.yellow(
+        chalk.bold(
+          'Note: If configued, migrations will trigger webhooks. You can disable webhooks on https://manage.sanity.io',
```

Comment:
I think this message is a little ambiguous about _what_ is configured:

1. The migration is configured to trigger webhooks (e.g. via a flag).
2. The dataset has configured webhooks.

Maybe this is clearer?

> If your dataset has active webhooks, migrations will trigger them.